src/language/ru-RU.json

```
@@ -934,13 +934,13 @@
     "select_accounts": "Выбранные аккаунты",
     "select_all": "Проверить Все",
     "select_contact": "Выберите контакт",
-    "select_custom_node": "Выбрать введённую ноду",
+    "select_custom_node": "Выбрать ноду",
     "select_opt_in_accounts": "Выбранные аккаунты для Опт-Ин процесса",
     "send": "Отправить",
     "sender": "Отправитель",
     "set_account_name": "Введите имя профиля",
     "set_default_explorer": "Введенный URL недействителен, для вас был установлен URL по умолчанию.",
-    "form_label_explorer_link": "Эксплорер URL",
+    "set_explorer_link": "Эксплорер URL",
```

```suggestion
    "set_explorer_link": "URL эксплорера",
```
Как думаешь, может лучше использовать `проводник` вместо `эксплорер`?

---

Эксплорер тоже не плохо и каких-либо трудностей в понимании не было, но если в будущем возникнут сложности, мы легко можем поменять его на `Проводник Блоков` или просто `Проводник`. Поэтому думаю можно пока оставить.